airbyte-integrations/bases/debezium-v1-9-6/src/main/java/io/airbyte/integrations/debezium/AirbyteDebeziumHandler.java

```diff
@@ -113,30 +106,20 @@ public AutoCloseableIterator<AirbyteMessage> getIncrementalIterators(final Confi
         publisher::close,
         firstRecordWaitTime);
 
-    // convert to airbyte message.
-    final AutoCloseableIterator<AirbyteMessage> messageIterator = AutoCloseableIterators
-        .transform(
-            eventIterator,
-            (event) -> DebeziumEventUtils.toAirbyteMessage(event, cdcMetadataInjector, emittedAt));
-
-    // our goal is to get the state at the time this supplier is called (i.e. after all message records
-    // have been produced)
-    final Supplier<AirbyteMessage> stateMessageSupplier = () -> {
-      final Map<String, String> offset = offsetManager.read();
-      final String dbHistory = trackSchemaHistory ? schemaHistoryManager
-          .orElseThrow(() -> new RuntimeException("Schema History Tracking is true but manager is not initialised")).read() : null;
-
-      return cdcStateHandler.saveState(offset, dbHistory);
-    };
-
-    // wrap the supplier in an iterator so that we can concat it to the message iterator.
-    final Iterator<AirbyteMessage> stateMessageIterator = MoreIterators.singletonIteratorFromSupplier(stateMessageSupplier);
-
-    // this structure guarantees that the debezium engine will be closed, before we attempt to emit the
-    // state file. we want this so that we have a guarantee that the debezium offset file (which we use
-    // to produce the state file) is up-to-date.
-
-    return AutoCloseableIterators.concatWithEagerClose(messageIterator, AutoCloseableIterators.fromIterator(stateMessageIterator));
+    Integer sync_checkpoint_seconds = config.get("sync_checkpoint_seconds") != null ? config.get("sync_checkpoint_seconds").asInt()
```

Comment:
this can be final, also the variable naming convention we follow is camel case so `syncCheckpointSeconds`

Reply:
My bad, modifying!